src/store/reducers/filter.ts

```diff
@@ -55,6 +55,7 @@ const FilterReducer = (
       return {
         ...state,
         filters: update(state.filters, { $push: [action.payload] }),
+        page: 1,
```

Comment:
missing isValid check here. We don't need to reset page when users add an empty filter